integration-auth/src/main/java/org/symphonyoss/integration/authentication/jwt/JwtAuthentication.java

```diff
@@ -57,10 +84,20 @@
   private IntegrationBridge integrationBridge;
 
   @Autowired
+  private IntegrationProperties properties;
+
+  @Autowired
   private IntegrationHttpApiClient integrationHttpApiClient;
 
+  @Autowired
+  private RsaKeyUtils rsaKeyUtils;
+
   private IntegrationAuthApiClient apiClient;
 
+  private PublicKey podPublicSignatureVerifier;
+
+  private Date podPublicSignatureVerifierExpirationDate;
```

Comment:
Why don't we use long instead of date? Maybe we could use Google Guava to manage the cache for us.